Jenkinsfile

```diff
@@ -331,6 +330,19 @@ pipeline {
                   script {
                     currentBuild.description = "${currentBuild.description?.trim() ? currentBuild.description : ''} released"
                   }
+                  archiveArtifacts(allowEmptyArchive: true, artifacts: "${env.BASE_DIR}/packages/rum/dist/bundles/*.js")
+                }
+              }
+            }
+            stage('Publish CDN') {
+              options { skipDefaultCheckout() }
+              steps {
+                script {
+                  def version = sh(label: 'Get package version', script: 'jq --raw-output .version packages/rum/package.json', returnStdout: true)
+                  publishToCDN(header: "Cache-Control:public,max-age=3600",
```

Comment:
Is this for testing? If not lets change the cache-control headers to be it for a year atleast `cache-control: public,max-age=31536000,immutable`

Reply:
```suggestion
                  publishToCDN(header: "Cache-Control:public,max-age=31536000,immutable",
```